gateway-package/src/main/resources/bin/start.sh

```diff
@@ -79,14 +79,15 @@
 # - ZWE_DISCOVERY_SERVICES_LIST
 # - ZWE_GATEWAY_SHARED_LIBS
 # - ZWE_haInstance_hostname
+# - ZWE_zowe_network_server_tls_attls
 # - ZWE_zowe_certificate_keystore_type - The default keystore type to use for SSL certificates
 # - ZWE_zowe_verifyCertificates - if we accept only verified certificates
 
 if [ -n "${LAUNCH_COMPONENT}" ]
 then
-    JAR_FILE="${LAUNCH_COMPONENT}/gateway-service-lite.jar"
+    JAR_FILE="${LAUNCH_COMPONENT}/gateway-service.jar"
```

Comment:
did you change the value in gateway-package/build.gradle as well?

Reply:
In the task download (`RegularFile gatewayJar = layout.projectDirectory.file("src/main/resources/bin/gateway-service.jar")`)? I am not sure how it is used...